src/thunderbots/software/test/ai/hl/stp/stp_refbox_game_state_play_selection.cpp

```diff
@@ -8,6 +8,181 @@
+     .ball_position      = Point(4.3, 2.8),
+     .ball_velocity      = Vector(),
+     .first_game_state   = RefboxGameState::DIRECT_FREE_US,
+     .second_game_state  = RefboxGameState::DIRECT_FREE_US},
+    {.name               = "Enemy direct free setup on friendly field side",
+     .friendly_positions = {{-4.0, 0}, {2.0, -3.0}, {-2.0, 0.0}, {-2.0, -1.0}},
+     .enemy_positions    = {{4.0, 0}, {-4.5, -3.0}, {2.0, 1.0}, {2.0, -1.0}},
+     .ball_position      = Point(-4.3, -2.8),
+     .ball_velocity      = Vector(),
+     .first_game_state   = RefboxGameState::DIRECT_FREE_THEM,
+     .second_game_state  = RefboxGameState::DIRECT_FREE_THEM},
+    {.name               = "Enemy direct free setup on enemy field side",
+     .friendly_positions = {{-4.0, 0}, {2.0, -3.0}, {-2.0, 0.0}, {-2.0, -1.0}},
+     .enemy_positions    = {{4.0, 0}, {4.5, 3.0}, {2.0, 1.0}, {2.0, -1.0}},
+     .ball_position      = Point(4.3, 2.8),
+     .ball_velocity      = Vector(),
+     .first_game_state   = RefboxGameState::DIRECT_FREE_THEM,
+     .second_game_state  = RefboxGameState::DIRECT_FREE_THEM},
+    {.name               = "Friendly penalty kick",
+     .friendly_positions = {{-4.0, 0}, {4.5 - 2.0, 0.0}, {0.3, 2.5}, {0.3, -2.5}},
+     .enemy_positions    = {{4.5, 0}, {0, 2.5}, {0, 0}, {0, -2.5}},
+     .ball_position      = Point(4.5 - 1.8, 0),
+     .ball_velocity      = Vector(),
+     .first_game_state   = RefboxGameState::PREPARE_PENALTY_US,
+     .second_game_state  = RefboxGameState::NORMAL_START},
+    {.name               = "Enemy penalty kick",
+     .friendly_positions = {{-4.5, 0}, {0, 0.0}, {0, 2.5}, {0, -2.5}},
+     .enemy_positions    = {{4.5, 0}, {-(4.5 - 2.0), 2.5}, {-0.3, 0}, {-0.3, -2.5}},
+     .ball_position      = Point(-(4.5 - 1.8), 0),
+     .ball_velocity      = Vector(),
+     .first_game_state   = RefboxGameState::PREPARE_PENALTY_THEM,
+     .second_game_state  = RefboxGameState::NORMAL_START},
+    {.name               = "Friendly penalty kick post",
+     .friendly_positions = {{-4.0, 0}, {4.5 - 2.0, 0.0}, {0.3, 2.5}, {0.3, -2.5}},
+     .enemy_positions    = {{4.5, 0}, {0, 2.5}, {0, 0}, {0, -2.5}},
+     .ball_position      = Point(4.5 - 1.0, 0),
+     .ball_velocity      = Vector(1, 0),
+     .first_game_state   = RefboxGameState::PREPARE_PENALTY_US,
+     .second_game_state  = RefboxGameState::NORMAL_START},
+    {.name               = "Enemy penalty kick post",
+     .friendly_positions = {{-4.5, 0}, {0, 0.0}, {0, 2.5}, {0, -2.5}},
+     .enemy_positions    = {{4.5, 0}, {-(4.5 - 2.0), 2.5}, {-0.3, 0}, {-0.3, -2.5}},
+     .ball_position      = Point(-(4.5 - 1.0), 0),
+     .ball_velocity      = Vector(-1, 0),
+     .first_game_state   = RefboxGameState::PREPARE_PENALTY_THEM,
+     .second_game_state  = RefboxGameState::NORMAL_START}};
+
+// INSTANTIATE_TEST_CASE_P(TestPositions,
+// STPRefboxGameStatePlaySelectionTestWithPositions,
+//                        ::testing::ValuesIn(test_params.begin(), test_params.end()));
```

Comment:
We should remove commented out code.

Reply:
ideally yes, but i wanted to make the change to enable the test very trivial once all plays are in place